charts/victoria-metrics-agent/values.yaml

```diff
@@ -16,7 +16,7 @@ deployment:
 
 # ref:  https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
 statefulset:
-  enabled: false
+  enabled: true
```

Comment:
Please revert this value as it brings breaking change.

Reply:
You are totally right, I forgot to reset that one. Is now at false again